components/carousel/carousel.scss

```diff
@@ -0,0 +1,86 @@
+.ui-carousel {
+  position: relative;
+  overflow: hidden;
+}
+
+.ui-carousel-track {
+  display: flex;
+  position: relative;
+  transition: var(--tx-carousel-track-transition);
+}
+
+.ui-carousel-item {
+  flex: 1 0 100%;
+  position: relative;
+  min-height: var(--tx-carousel-item-min-height);
+}
+
+.ui-carousel-item-image {
```

Comment:
I believe this shouldn't be something we can talk about, I'm fine with non-sorted alphabetically, but since we have it in many other repos, would be nice to be consistent. But I also believe that this could be automated by something like this: https://github.com/morishitter/stylefmt